R/MixedModelsCommon.R

```diff
@@ -792,1293 +793,4580 @@
+      "The Hamiltonian Monte Carlo procedure might be inefficient -- %i transitions exceeded the maximum tree depth. This can be solved by carefully increasing 'Maximum tree depth"
+    ),
+    iterations
+  )
+}
+.mmMessageMaxRhat       <- function(Rhat) {
+  gettextf(
+    "Inference possibly unreliable -- MCMC chains might not have converged; The largest R-hat is %.3f > 1.01. To lower R-hat please increase 'Iterations', or 'Adapt delta' in the Options section.",
+    Rhat
+  )
+}
+.mmMessageMinESS        <- function(ESS, treshold) {
+  gettextf(
+    "Low estimation accuracy -- The smallest Effective Sample Size (ESS) is %.2f < %1.0f. To increase accuracy please increase 'Iterations', or 'Adapt delta' in the Options section.",
+    ESS,
+    treshold
+  )
+}
+.mmMessageBadWAIC       <- function(n_bad) {
+  sprintf(
+    ngettext(
+      n_bad,
+      "WAIC estimate unreliable -- There was %1.0f p_waic estimate larger than 0.4. We recommend using LOO instead.",
+      "WAIC estimate unreliable -- There were %1.0f p_waic estimates larger than 0.4. We recommend using LOO instead."
+    ),
+    n_bad
+  )
+}
+.mmMessageBadLOO        <- function(n_bad) {
+  sprintf(
+    ngettext(
+      n_bad,
+      "LOO estimate unreliable -- There was %1.0f observation with the shape parameter (k) of the generalized Pareto distribution higher than > .5.",
+      "LOO estimate unreliable -- There were %1.0f observations with the shape parameter (k) of the generalized Pareto distribution higher than > .5."
+    ),
+    n_bad
+  )
+}
+.mmMessageFitType       <- function(REML) {
+  gettextf("The model was fitted using %1$s.%2$s",
+           ifelse(REML, gettext("restricted maximum likelihood"), gettext("maximum likelihood")),
+           ifelse(REML, gettext(" Please note that models with different fixed effects cannot be compared when REML is used. To use ML, switch 'Test model terms' to 'Likelihood ratio tests'."), ""))
+}
+#
+# Copyright (C) 2019 University of Amsterdam
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+# TODO: Expose priors specification to users in Bxxx?
+# TODO: Add 3rd level random effects grouping factors ;) (not that difficult actually)
+
+.mmRunAnalysis   <- function(jaspResults, dataset, options, type){
```

Comment:
what happened here? did you accidentally duplicate the analysis?

Reply:
no, I was just removing the previously present underscores Lexi complained about. (unless you see the same file twice there, but I see only one copy)

Reply:
oh, now I see, I though it was line 1

Reply:
yeah, wtf happened here? thanks for finding it out